Src/IronPython.Modules/zipimport.cs

```diff
@@ -212,13 +212,13 @@ private enum ModuleStatus {
                 PythonContext pythonContext = PythonContext.GetContext(context);
                 PythonDictionary dict;
                 ScriptCode script = null;
-                string code = GetModuleCode(context, fullname, out ispackage, out modpath);
+                byte[] code = GetModuleCode(context, fullname, out ispackage, out modpath);
                 if (code == null) {
                     return null;
                 }
 
                 mod = pythonContext.CompileModule(modpath, fullname,
-                    new SourceUnit(pythonContext, new SourceStringContentProvider(code), modpath, SourceCodeKind.File),
+                    new SourceUnit(pythonContext, new MemoryStreamContentProvider(context.GetPythonContext(), code, modpath), modpath, SourceCodeKind.File),
```

Comment:
Hmm, another way to get the language context? Didn't know about this one. Maybe use `pythonContext` instead since we already have it.